### PR TITLE
blogpost Raise_some_horns_Red_Hat_s_MetalKube_aims_to_make_Kubernetes…

### LINKED ISSUE #73: convert media references on documentation page to blog posts
Instead of providing a list of links, we could create small blog posts linking to each of the articles. Maybe not one per article, since several of them were on the same day.

### PR DESCRIPTION
**Pull request by ptrnull**
Blogpost  Raise_some_horns_Red_Hat_s_MetalKube_aims_to_make_Kubernetes_on_bare_machines_simple

Let's publish it next week (Tuesday 17th is good, I'll let it in hold, you can lgtm/approve if looks good)
Closes #73 
Signed-off-by: Pedro Ibáñez <pedro@redhat.com>

### CODE REVIEW

File: _posts/2019-04-12-Raise_some_horns_Red_Hat_s_MetalKube_aims_to_make_Kubernetes_on_bare_machines_simple.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+The Open-source software developers at Red Hat are working on a tool that would simplify deployment and management of Kubernetes clusters on bare-metal servers.
+
+Steve told The reg:
+> "In some situations, you won't want to run a full OpenStack infrastructure-as-a-service layer to provide, potentially, for multiple Kubernetes clusters".
+
+Hardy is a notable contributor to OpenStack, having previously worked on Heat and TripleO projects. He said one of the reasons for choosing Ironic was its active development – and when new features get added to Ironic, the Metal³ team gets them "for free".
+
+> "OpenStack has always been a modular set of projects, and people have always had the opportunity to reuse components for different applications. This is just an example of where we are leveraging one particular component for infrastructure management, just as an alternative to using a full infrastructure API," Hardy said.
+
+Thierry Carrez, veep of engineering at the OpenStack Foundation also told The Reg:
```
Comment (russellb) [inline _posts/2019-04-12-Raise_some_horns_Red_Hat_s_MetalKube_aims_to_make_Kubernetes_on_bare_machines_simple.md]: VP?

**Comment by russellb** [comment]
minor nit, but otherwise lgtm.  feel free to merge when you're ready

File: _posts/2019-04-12-Raise_some_horns_Red_Hat_s_MetalKube_aims_to_make_Kubernetes_on_bare_machines_simple.md (hunk 1 above)
Comment (ptrnull) [reply _posts/2019-04-12-Raise_some_horns_Red_Hat_s_MetalKube_aims_to_make_Kubernetes_on_bare_machines_simple.md]: "veep" is a copy&paste from the original article:

> "I like the fact that the projects end up being reusable on their own, for the functions they bring to the table – this helps us integrate with adjacent communities," Thierry Carrez, veep of engineering at the OpenStack Foundation, told The Reg.

Never read it before, and yes, it's VP.

**Comment by ptrnull** [comment]
> minor nit, but otherwise lgtm. feel free to merge when you're ready

Thanks @russellb !

**Pull request merged**